packages/@okta/courage-for-signin-widget/src/CourageForSigninWidget.js

```diff
@@ -103,8 +117,8 @@ const Okta = {
         },
 
         inputs: {
-          TextBox,
-          PasswordBox,
+          TextBox: TextBoxForSigninWidget,
```

Comment:
This will apply here too right? https://github.com/okta/okta-signin-widget/blob/master/src/views/shared/TextBox.js#L20 
Just checking that no changes needed in local TextBox.js.

Reply:
Yes, absolutely.